src/api.rs

```diff
@@ -140,6 +140,37 @@ impl Context {
     Ok(())
   }
 
+  pub fn container_sequence_header(&mut self) -> Vec<u8> {
+    use bitstream_io::*;
+    use std::io;
+    fn sequence_header_inner(seq: &Sequence) -> io::Result<Vec<u8>> {
```

Comment:
I think I encounter the naming '_inner' second time here, with the first one found in intra prediction?
Could you explain what it would mean? I am just curious on what the name means here... :)

Reply:
In general when I have some function I can model with some internal function called only from the actual API function I tend to use `_inner`.

In this code I'm hiding the fact bitstream-io focus is on writing on generic I/O, thus accounting for failures that cannot happen if you are writing to memory.

So instead of writing lots of `unwrap()` to each `write()` function I just use the `?` operator and unwrap the return value.